stock_exception/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Stock Exception",
     "summary": "Custom exceptions on stock picking",
-    "version": "16.0.1.0.0",
+    "version": "16.0.1.1.0",
```

Comment:
don't include this change unless you have migration scripts. Oca bot will do it.